rancher2/schema_cluster_rke_config_network.go

```diff
@@ -26,6 +28,273 @@ var (
 
 //Schemas
 
+func clusterRKEConfigNetworkAciFields() map[string]*schema.Schema {
+	s := map[string]*schema.Schema{
+		"aep": {
```

Comment:
The following fields need to be required/sensitive/optional as specified in the [rke_cluster](https://registry.terraform.io/providers/rancher/rke/latest/docs/resources/cluster) resource docs. To make a field Required/Sensitive, add `Sensitive: true.` We want to make sure that if a user specifies an `aci_network_provider` configuration, the necessary fields are there to prevent provisioning errors when rke is deployed.

Required/Sensitive: `token, apic_user_key, apic_user_crt`

Required: `system_id, apic_hosts, apic_user_name, encap_type, mcast_range_start, mcast_range_end, aep, vrf_name, vrf_tenant, l3out, node_subnet, l3out_external_networks, extern_dyanmic, extern_static, node_svc_subnet, kube_api_vlan, service_vlan`

Optional: `snat_port_range_start, snat_port_range_end, snat_ports_per_node`